browser/src/Services/Learning/Tutorial/Tutorials/TargetsVimPluginTutorial.tsx

```diff
@@ -0,0 +1,142 @@
+
+const Line5a = "Replacing the text character with 'a' will find the next programming argument."
+const Line5b = "'dina' will select inside the (next, argument)."
+const Line5c = "'dana' selects around the (next, argument)"
+
+const Line6a = "These are only a brief overview of Targets.vim."
+const Line6b = "More advanced features can be found on its github repository."
+const Line6c = "https://github.com/wellle/targets.vim"
+
+export class TargetsVimPluginTutorial implements ITutorial {
+    private _stages: ITutorialStage[]
+
+    constructor() {
+        this._stages = [
+            new Stages.SetBufferStage([Line1a, Line1b, Line1c]),
+            new Stages.SetCursorPositionStage(0, 0),
+            new Stages.MoveToGoalStage("Use 'cin(' to move inside the paranthesis", 2, 46),
+            new Stages.WaitForModeStage("Ready to type.  Go back to 'normal' mode", "normal"),
+            Stages.combine(
+                null,
+                new Stages.FadeInLineStage(null, 1, Line2a),
+                new Stages.FadeInLineStage(null, 2, Line2b),
+                new Stages.SetBufferStage([Line2a, Line2b, Line2c]),
+            ),
+            new Stages.SetCursorPositionStage(0, 7),
+            new Stages.MoveToGoalStage("Use 'cin(' to move inside the paranthesis", 0, 40),
+            new Stages.WaitForModeStage("Ready to type.  Go back to 'normal' mode", "normal"),
+            new Stages.SetCursorPositionStage(1, 7),
+            new Stages.MoveToGoalStage("Use 'can(' to move around the paranthesis", 1, 39),
+            new Stages.WaitForModeStage("Ready to type.  Go back to 'normal' mode", "normal"),
+            new Stages.MoveToGoalStage(
+                "Use 'cil(' to move back inside the last paranthesis",
+                0,
+                40,
+            ),
+            new Stages.WaitForModeStage("Ready to type.  Go back to 'normal' mode", "normal"),
+            Stages.combine(
+                null,
+                new Stages.FadeInLineStage(null, 1, Line3a),
+                new Stages.FadeInLineStage(null, 2, Line3b),
+                new Stages.SetBufferStage([Line3a, Line3b, Line3c]),
+            ),
+            new Stages.SetCursorPositionStage(0, 0),
+            new Stages.MoveToGoalStage("Use 'cin'' to move inside the single quotes", 0, 19),
+            new Stages.WaitForModeStage("Ready to type.  Go back to 'normal' mode", "normal"),
+            new Stages.SetCursorPositionStage(1, 7),
+            new Stages.MoveToGoalStage(
+                "Use 'cIn\"' to move to first character inside double quotes",
+                1,
+                48,
+            ),
+            new Stages.WaitForModeStage("Ready to type.  Go back to 'normal' mode", "normal"),
+            new Stages.SetCursorPositionStage(2, 7),
+            new Stages.MoveToGoalStage("Use 'cAn\"' to select around the double quotes", 2, 26),
+            new Stages.WaitForModeStage("Ready to type.  Go back to 'normal' mode", "normal"),
```

Comment:
I realized at first when I got to this step - I wasn't sure if I was  supposed to do anything - I think the 'Ready to type.' was a bit ambigious (for me). We could consider making it a bit more explicit, like:
- "Press ESC to go back to 'normal' mode."

Or perhaps just:
- "Go back to 'normal' mode"


Or have them actually enter some text:
- "Type 'foo'."
- " Press ESC to go back to normal mode."